hnn_core/io.py

```diff
@@ -0,0 +1,338 @@
+"""Network io"""
+
+# Authors: Rajat Partani <rajatpartani@gmail.com>
+#          Mainak Jas <mjas@mgh.harvard.edu>
+#          Nick Tolley <nicholas_tolley@brown.edu>
+
+import os
+from h5io import write_hdf5, read_hdf5
+
+from .cell import Cell, Section
+from .cell_response import CellResponse
+from .externals.mne import fill_doc
+
+
+def str_to_node(node_string):
+    node_tuple = node_string.split(',')
+    node_tuple[1] = int(node_tuple[1])
+    node = (node_tuple[0], node_tuple[1])
+    return node
+
+
+@fill_doc
+def write_network(net, fname, overwrite=True, save_unsimulated=False):
```

Comment:
you need to change `save_unsimulated`

Reply:
What was the name we decided? `save_output`?

Reply:
Then accordingly `read_raw` will become `read_output`?

Reply:
yep, exactly! `save_output` and `read_output`